src/third_party/graphlite/include/graph_lite.h

```diff
@@ -1,9 +1,11 @@
-// From https://github.com/haasdo95/graphlite commit 9cd2815e5d571a87e5b8b8ef3752e04d971f35d4
+// From https://github.com/haasdo95/graphlite commit
+// 26b9d8c05b5b2a93def516a1c50857695e8efe06
 #ifndef GSK_GRAPH_LITE_H
 #define GSK_GRAPH_LITE_H
 
 #include <algorithm>
 #include <cassert>
+#include <ciso646>  // Added to fix MSVC error
```

Comment:
There's not actually much change here - I think it's just formatting. I added this line to make MSVC able to compile `and` in addition to `&&`. The only update I pulled in was to not emit warnings about duplicate edges.

Reply:
Interesting! According to cppreference

> This means that in a conforming implementation, including this header has no effect. 

Thought MSVC is a conforming implementation :P